R/CreatePackageReport.R

```diff
@@ -51,6 +52,9 @@ CreatePackageReport <- function(pkg_name
       , pkg_name = pkg_name
     )
     
+    # Open Report
+    if (auto_open_report) {browseURL(report_path)}
```

Comment:
Want to lodge my opinion that this should be on its own line

```
if (auto_open_report) {
    browseURL(report_path)
}
```

...but it is not a hill I'm willing to die on